spit/classifier.py

```diff
@@ -1,110 +1,55 @@
-        self.init_session()
-        self.init_variables()
-        self.init_saver()
-        # Load?
-        if croot is not None:
-            print("Loading the Classifier: {:s}".format(croot))
-            self.saver.restore(sess=self.session, save_path=croot)
-
-    def init_session(self):
-        """ Initialize a Tensorflow session
-        """
-        self.x = tf.placeholder(tf.float32, shape=[None, self.preproc_dict['img_size_flat']], name='x')
-        x_image = tf.reshape(self.x, [-1, self.preproc_dict['image_height'],
-                                      self.preproc_dict['image_width'],
-                                      self.preproc_dict['num_channels']])
-        self.y_true = tf.placeholder(tf.float32, shape=[None, len(self.label_dict)], name='y_true')
-        self.y_true_cls = tf.argmax(self.y_true, axis=1)
-        x_pretty = pt.wrap(x_image)
-
-        # Architecture
-        with tf.Graph().as_default(), pt.defaults_scope(activation_fn=tf.nn.relu):
-            self.y_pred, loss = x_pretty. \
-                conv2d(kernel=5, depth=36, name='layer_conv1'). \
-                max_pool(kernel=2, stride=2). \
-                conv2d(kernel=5, depth=64, name='layer_conv2'). \
-                max_pool(kernel=2, stride=2). \
-                flatten(). \
-                fully_connected(size=128, name='layer_fc1'). \
-                softmax_classifier(num_classes=len(self.label_dict), labels=self.y_true)
-
-        self.optimizer = tf.train.AdamOptimizer(learning_rate=1e-4).minimize(loss)
-        self.y_pred_cls = tf.argmax(self.y_pred, axis=1)
-        self.correct_prediction = tf.equal(self.y_pred_cls, self.y_true_cls)
-        self.accuracy = tf.reduce_mean(tf.cast(self.correct_prediction, tf.float32))
-        self.session = tf.Session()
-        # Return
-        return
-
-    def init_variables(self):
-        """ Initialize variables
-        """
-        self.session.run(tf.global_variables_initializer())
-        return
-
-    def init_saver(self):
-        self.saver = tf.train.Saver()
+  
+  def __init__(self, label_dict, preproc_dict, classify_dict, **kwargs):
+    self.label_dict = label_dict.copy()
+    self.preproc_dict = preproc_dict.copy()
+    # is this for prediction? <----
+    self.classify_dict = classify_dict.copy() 
+    
+    # Set up tensorflow/keras model
+    self.init_session()
+    
+  
+  # One major change from tf1.0 to tf2.0
+  # is that tf.layers module no longer relies on tf.variable_scope
+  # and no more use of tf.placeholder
+  # no more session.run() although not actual functioning originally 
+  def init_session(self):
+    
+    # building a simple model
+    # linear stack of layers
+
+    self.model = keras.Sequential([
+        # 2D convolution followed by a maxpool, change data format when actual dataset etc. comes or when testing
+        keras.layers.Conv2D(36, kernel_size=5, strides=(1, 1), padding='valid', activation='relu', input_shape = (210, 650, 1)),
```

Comment:
As in the previous, please have `input_shape` be set by the `preproc_dict`, e.g.
```
self.preproc_dict['image_height'],	
                                      self.preproc_dict['image_width'],	
                                      self.preproc_dict['num_channels']
```